plugins/catalog/src/components/AboutCard/AboutCard.tsx

```diff
@@ -170,7 +170,11 @@ export function AboutCard(props: AboutCardProps) {
           })),
   };
 
-  const subHeaderLinks = [viewInSource, viewInTechDocs];
+  const subHeaderLinks = [viewInSource];
+
+  if (!viewInTechDocs?.disabled) {
```

Comment:
I would suggest this instead:

```suggestion
  if (viewTechdocLink) {
```

so that we still display the TechDocs link in case TechDocs is configured but the selected component doesn't have anything.